src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
```

Comment:
Ensure that the element with the class 'logo' exists in the DOM. If it doesn't, `logo` will be `null`, leading to an error when trying to add an event listener.